### ISSUE TITLE
Redesign Form Labels & Ensure Consistent Usage Throughout dotCMS

### ISSUE THREAD

**Issue by damen-dotcms** [dotCMS/core#23635]
## Problem Statement
Labels in  edit a content type are not consistent across the form.

Label not consistent, some of them in black 14px, the others light gray 12px. Accessibility issues as well.

[![Screenshot_2022-12-21_15-22-46.png](https://mrkr.io/s/63a324c6d1b8a94b47159062/2)](https://mrkr.io/s/63a324c6d1b8a94b47159062/0)

## Steps to Reproduce
1. Go to Content Types
2. Select a content type
3. Click edit
4. See the error

## Additional context:
We need to do some extra clean-up here, so...

1. We can remove the `.form__label` and use `.field label` to style
2. We need to make the `*` red (see the `variable.scss` file)
3. Check system-wide the forms and make sure we didn't break anything.

## Acceptance Criteria
Users should see consistent labels, so this might be required some research, things I know:

---

**Reported by:** Melissa Rojas Rodríguez (melissa.rojas@dotcms.com)

**Source URL:** [https://demo.dotcms.com/dotAdmin/#/content-types-angular/create/content](https://demo.dotcms.com/dotAdmin/#/content-types-angular/create/content)

**Issue details:** [Open in Marker.io](https://app.marker.io/i/63a324c6d1b8a94b47159065_f856a6eab78a69f9?advanced=1)

<table><tr><td><strong>Device type</strong></td><td>desktop</td></tr><tr><td><strong>Browser</strong></td><td>Chrome 108.0.0.0</td></tr><tr><td><strong>Screen Size</strong></td><td>1440 x 900</td></tr><tr><td><strong>OS</strong></td><td>OS X 10.14.6</td></tr><tr><td><strong>Viewport Size</strong></td><td>1440 x 821</td></tr><tr><td><strong>Zoom Level</strong></td><td>100%</td></tr><tr><td><strong>Pixel Ratio</strong></td><td>@&#8203;2x</td></tr></table>

**Comment by Melissa-dotCMS** [comment]
All labels should be black, 14px, and regular (with no bold).

**Comment by fmontes** [comment]
### List of forms to check
1. Apps
2. Block editor settings
3. Experiments
4. Content types edit
5. Content types fields
6. Content types add to menu
7. Content type > copy
8. Download bundle dialog
9. Push Publish form
10. Create persona (from edit page)
11. Template properties
12. My Account
13. Login
14. Forgot Password
15. Reset Password

**Comment by josemejias07** [comment]
**Failed QA** - Tested on **23.05_3dccd573_SNAPSHOT** // Docker // macOS 13.0 // FF v111.0

Not in docker
![Screenshot 2023-04-03 at 10 41 16 AM](https://user-images.githubusercontent.com/110418590/229573689-7455d8df-6e88-4da6-8020-ef1e297b058c.png)

**Comment by fmontes** [comment]
Sorry I send this to QA by mistake.

**Comment by damen-dotcms** [comment]
I'm still seeing quite a few labels that don't line up. Should I report them here or in new tickets?

**Comment by alfredo-dotcms** [comment]
still needs work


![Image](https://github.com/dotCMS/core/assets/37185433/67bece15-0e9d-4048-b9b4-65d08eb0411e)

**Comment by alfredo-dotcms** [comment]
LGTM!


![Image](https://github.com/dotCMS/core/assets/37185433/da816c2a-04af-4e16-bb2a-c3c9daa49682)

**Comment by josemejias11** [comment]
**Approved QA** - Tested on **master_ae620c0_SNAPSHOT** // Docker // macOS 13.0 // FF v113.0
